docs/.vuepress/config.js

```diff
@@ -58,25 +58,25 @@ const blogSidebar = [
 
 module.exports = {
   title,
-  description: 'LDK is a flexible lightning implementation with supporting batteries (or modules).',
+  description: 'PDK is a Payjoin implementation with supporting modules.',
   theme: resolve(__dirname, '../../node_modules/@spiralbtc/vuepress-devkit-theme'),
   ...themeConfig({
     baseUrl,
     title,
     themeColor,
-    tags: ['Bitcoin', 'Lightning', 'LDK', 'Lightning Dev Kit', 'Documentation']
+    tags: ['Bitcoin', 'Payjoin', 'PDK', 'Payjoin Dev Kit', 'Documentation']
   }),
   themeConfig: {
     domain: baseUrl,
     logo: '/img/logo.svg',
     displayAllHeaders: false,
-    repo: 'lightningdevkit/lightningdevkit.org',
+    repo: 'payjoin/payjoindevkit.org',
     docsDir: 'docs',
     docsBranch: 'main',
     editLinks: true,
     sidebarDepth: 0,
     algolia: {
-      indexName: 'lightningdevkit',
+      indexName: 'payjoindevkit',
       appId: 'BH4D9OD16A',
       apiKey: '17ed8a4e16a1cb7d94da4e96f2ff817f',
```

Comment:
It's unclear to me where these values are used and whether they should be changed.

Reply:
This is for algolia text search in the original lightningdevkit.org. I'm not sure if it's working as-is or if we need to register our own app

https://github.com/lightningdevkit/lightningdevkit.org/issues/58

Reply:
Oh yeah, currently the search is actually displaying results from lightningdevkit.org.

<img width="492" alt="Screenshot 2024-04-19 at 10 24 21" src="https://github.com/payjoin/payjoindevkit.org/assets/144076611/5367e699-70db-4bb0-8b22-c90a5102e587">

Reply:
Looks like we need an algolia profile